views/incorrect-information/wrong-ro.html

```diff
@@ -1,23 +1,101 @@
+                <span class="js-step-title-text">
+                  Update your registered office address
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item">
+                  <p>You will need to
+                    <a href="/change-data/change-address" target="_blank">update the registered office address (opens in a new tab)</a>
+                    before you can file a confirmation statement.</p>
+                </li>
+              </ol>
+            </div>
+          </li>
+
+          <li class="app-step-nav__step js-step">
+            <div class="app-step-nav__header js-toggle-panel" data-position="2">
+              <h2 class="app-step-nav__title">
+                <span class="app-step-nav__circle app-step-nav__circle--number">
+                  <span class="app-step-nav__circle-inner">
+                    <span class="app-step-nav__circle-background">
+                      <span class="govuk-visually-hidden">Step</span> 2
+                    </span>
+                  </span>
+                </span>
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button"/>
+                </span>
+                <span class="js-step-title-text">
+                  Return to this service
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item ">
+                  <p>Once you have updated the registered office address, you will need to close that tab to return to this screen.</p>
+                </li>
+              </ol>
+            </div>
+          </li>
+
+          <li class="app-step-nav__step js-step">
+            <div class="app-step-nav__header js-toggle-panel" data-position="3">
+              <h2 class="app-step-nav__title">
+                <span class="app-step-nav__circle app-step-nav__circle--number">
+                  <span class="app-step-nav__circle-inner">
+                    <span class="app-step-nav__circle-background">
+                      <span class="govuk-visually-hidden">Step</span> 3
+                    </span>
+                  </span>
+                </span>
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button"/>
+                </span>
+                <span class="js-step-title-text">
+                  Complete your confirmation statement
+                </span>
+              </h2>
+            </div>
+            <div>
+              <p>Return to the <a href="/confirmation-statement/ro">Registered office address</a> section and confirm that an update has been submitted.</p>
```

Comment:
this href-href="/confirmation-statement/ro" doesn't exist in our app, you'll need to find out what screen they want to return to and test the link in your running docker env